sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -520,6 +520,12 @@ def _add_argparse_args(cls, parser):
         type=str,
         help='GCE minimum CPU platform. Default is determined by GCP.'
     )
+    parser.add_argument(
+        '--dataflow_worker_jar',
+        dest='dataflow_worker_jar',
+        type=str,
+        help='Dataflow worker jar.'
```

Comment:
Could you update the description here. 

We would not expect users to use this option typically. Biggest use case is probably development related changes. And it also cannot be used for legacy pipelines either. (Should this be an error, if fn api experiment is not set but this flag is used?)